### PR TITLE
Added Array type to navigateTo prop type check

### PR DESCRIPTION
**Pull request by rico-ocepek**
## Description
Changed the property type of the carousel's navigateTo property from Number only to [Number, Array].
Also updated readme.md to describe the new behaviour.
Updated the "NavigateTo slides" Vue play example to include an example of the new API usage.

## Motivation and Context
As the pending fix from [https://github.com/SSENSE/vue-carousel/issues/257](url) should finally be done I decided to help you out a bit.

## How Has This Been Tested?
As the feature was already implemented and only the property type was wrong further tests should not be required.
However I did test both cases of the navigateTo property (animated and non animated) and couldn't spot any errors.
As I said I also included an example in the Vue play file for testing purposes.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have included a vue-play example (if this is a new feature)

### CODE REVIEW

File: src/Carousel.vue [hunk 1]
```diff
@@ -177,7 +177,7 @@ export default {
      * Listen for an external navigation request using this prop.
      */
     navigateTo: {
-      type: Number,
+      type: [Number, Array],
```
Comment (adin234) [inline src/Carousel.vue]: can't we do 
```json
type: Number | [Number, Array],
```

Comment (rico-ocepek) [reply src/Carousel.vue]: Hi Adin, could you explain what this would do / improve?

**Comment by quinnlangille** [comment]
Hey @rico-ocepek, thanks for this! I'll test and merge in this afternoon 🚀

**Comment by rico-ocepek** [comment]
Hi @quinnlangille, sorry to bother you again - do we have any progress here? Would love to integrate this feature in a client project without anyone asking why there is an error message 😅

**Comment by quinnlangille** [comment]
Hey @rico-ocepek sorry for the delay! I've merged and published as an alpha tag since you needed it quickly - you can find it with `yarn install vue-carousel@0.18.0-alpha` 🚀

**Comment by rico-ocepek** [comment]
No worries @quinnlangille, thank you for your help and your great work here! 👍🏻

**Pull request merged**